src/newcharacter.cpp

```diff
@@ -2419,10 +2419,14 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
     select_location.entries.emplace_back( entry_random_start_location );
     for( const auto &loc : start_locations::get_all() ) {
         if( get_scenario()->allowed_start( loc.id ) ) {
-            uilist_entry entry( loc.id.get_cid().to_i(), true, -1, string_format(
-                                    ngettext( "%s (<color_white>%d</color> variant)",
-                                              "%s (<color_white>%d</color> variants)",
-                                              loc.targets_count() ), loc.name(), loc.targets_count() ) );
+            std::string loc_name = _( loc.name() );
```

Comment:
```suggestion
            std::string loc_name = loc.name();
```
`loc.name()` is already translated, no need to translated twice.